website/blog/2022-05-10-faster-presto-queries-with-parquet-page-index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Faster Presto Queries with Parquet Page Index
+author: Xinli Shang
+authorURL: https://www.linkedin.com/in/xinli-shang-2479863/
+---
+**Uber:** Xinli Shang
+
+**Uber:** Chen Liang
+
+## Introduction
+Today’s data is growing very fast, which creates challenges for query engines like Presto.  Presto is a popular interactive query engine, because of its scalability, high performance, and smooth integration with Hadoop. As the volume of data grows, Presto needs to read larger chunks of data and load them into memory, which causes higher IO and memory usage and GC time etc.
+Apache Parquet is an open source, column-oriented data file format designed for efficient data storage and retrieval. It provides efficient data compression and encoding schemes with enhanced performance to handle complex data in bulk.
```

Comment:
This doesn't render as a paragraph when you create it. Maybe add a newline in between paragraphs?